nexus.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+var fluid = require("infusion"),
+    kettle = fluid.registerNamespace("kettle"),
+    path = require("path");
+
+require("kettle");
+require("./src/Nexus.js");
+
+kettle.config.loadConfig({
+    configName: kettle.config.getNodeEnv("gpii.nexus.config"),
+    configPath: kettle.config.getConfigPath(path.resolve(__dirname, "./configs"))
```

Comment:
With the above directive, this can just be "%gpii-nexus/configs" as per https://github.com/amb26/kettle/blob/KETTLE-32/lib/KettleConfigLoader.js#L48